content/en/docs/setup/install/operator/index.md

```diff
@@ -217,20 +217,20 @@ for the complete set of configuration settings.
 Download and extract the `istioctl` corresponding to the version of Istio you wish to upgrade to. Reinstall the operator
 at the target Istio version:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=none >}}
```

Comment:
Seems kind of misleading to mark this document as "tested" when none of the upgrade parts are tested.

Can you maybe uninstall Istio in the test at this point, and then reinstall using the previous minor version, and then you could test upgrade to the current version? I think that should be possible, but a bit of extra work.

Reply:
this was addressed in 6941f93a050425a64dfd08cdd45cf8452587b22c